src/components/Gallery/Gallery.stories.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Gallery from './'
+import * as directors from '../../directors/directors.json'
+import { withKnobs, boolean, select, number } from '@storybook/addon-knobs'
+
+const images = directors.directors[1].gallery
+const imagesOptions = images.reduce((acc, currValue, index) => {
```

Comment:
const imagesOptions = images.reduce((acc, currValue, index) =>({ ...acc, [currValue]: index }), {})

Reply:
done, I'm too used to my "return-style" )